app/controllers/users_controller.rb

```diff
@@ -23,6 +22,10 @@ def update_success
     redirect_to action: 'settings'
   end
 
+  def settings_params
+    params.permit(student: [:receives_invite_emails, :receives_results_emails, :display_name])
```

Comment:
Rather than making this nested, use `.require(:student).permit(...)` (you can see similar examples throughout the controller code). This will also allow you to pass `settings_params` directly to the `update!` call above.